README.md

```diff
@@ -1 +1,18 @@
 # Node.js Inclusivity Slack Inviter
+
+Automated [Slack](https://slack.com/) invitations for the [Node.js Inclusivity WG](https://github.com/nodejs/inclusivity).
+
+## Configuration
+
+Configuration settings are stored in `config.json`. The following settings must be defined:
```

Comment:
I tend to prefer config files myself, although env vars can be more flexible in certain CI/deployment scenarios